diff --git a/node_modules/react-native/scripts/cocoapods/helpers.rb b/node_modules/react-native/scripts/cocoapods/helpers.rb
index 5ef30a3..04c18d5 100644
--- a/node_modules/react-native/scripts/cocoapods/helpers.rb
+++ b/node_modules/react-native/scripts/cocoapods/helpers.rb
@@ -38,7 +38,7 @@ end
 module Helpers
     class Constants
         def self.min_ios_version_supported
-            return '13.4'
+            return '14.0'
         end
 
         def self.min_xcode_version_supported
