diff --git a/node_modules/react-native/Libraries/Lists/FlatList.js b/node_modules/react-native/Libraries/Lists/FlatList.js
index f34fe50..fca6c3a 100644
--- a/node_modules/react-native/Libraries/Lists/FlatList.js
+++ b/node_modules/react-native/Libraries/Lists/FlatList.js
@@ -424,6 +424,7 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
 
   constructor(props: Props<ItemT>) {
     super(props);
+    this.props = props;
     this._checkProps(this.props);
     if (this.props.viewabilityConfigCallbackPairs) {
       this._virtualizedListPairs =
diff --git a/node_modules/react-native/scripts/cocoapods/helpers.rb b/node_modules/react-native/scripts/cocoapods/helpers.rb
index 5ef30a3..04c18d5 100644
--- a/node_modules/react-native/scripts/cocoapods/helpers.rb
+++ b/node_modules/react-native/scripts/cocoapods/helpers.rb
@@ -38,7 +38,7 @@ end
 module Helpers
     class Constants
         def self.min_ios_version_supported
-            return '13.4'
+            return '14.0'
         end
 
         def self.min_xcode_version_supported
