diff --git a/node_modules/react-native-gesture-handler/lib/commonjs/handlers/gestures/nativeGesture.ts b/node_modules/react-native-gesture-handler/lib/commonjs/handlers/gestures/nativeGesture.ts
index b8b00ab..9c5a1ce 100644
--- a/node_modules/react-native-gesture-handler/lib/commonjs/handlers/gestures/nativeGesture.ts
+++ b/node_modules/react-native-gesture-handler/lib/commonjs/handlers/gestures/nativeGesture.ts
@@ -4,6 +4,7 @@ import type { NativeViewGestureHandlerPayload } from '../GestureHandlerEventPayl
 
 export class NativeGesture extends BaseGesture<NativeViewGestureHandlerPayload> {
   public config: BaseGestureConfig & NativeViewGestureConfig = {};
+  public handlerName: string;
   
   constructor() {
     super();
