diff --git a/node_modules/react-native-date-picker/index.d.ts b/node_modules/react-native-date-picker/index.d.ts
index 0ab796f..369b024 100644
--- a/node_modules/react-native-date-picker/index.d.ts
+++ b/node_modules/react-native-date-picker/index.d.ts
@@ -98,6 +98,9 @@ export interface DatePickerProps extends ViewProps {
   /** Color of the divider on Android */
   dividerColor?: string
 
+  /** Color of the date picker text */
+  textColor?: string
+
   /** Color of the android modal buttons */
   buttonColor?: string
 }
diff --git a/node_modules/react-native-date-picker/src/index.js b/node_modules/react-native-date-picker/src/index.js
index 90673e9..9207fd9 100644
--- a/node_modules/react-native-date-picker/src/index.js
+++ b/node_modules/react-native-date-picker/src/index.js
@@ -55,6 +55,9 @@ const getDividerColor = (props) => {
 
 /** @param {Props} props **/
 const getTextColor = (props) => {
+  const _textColor = props.textColor;
+  if (_textColor != null) return _textColor;
+  
   const theme = getTheme(props)
   if (theme === 'dark') return 'white'
   if (theme === 'light') return 'black'
