diff --git a/node_modules/react-native-gifted-charts/src/BarChart/Animated2DWithGradient.tsx b/node_modules/react-native-gifted-charts/src/BarChart/Animated2DWithGradient.tsx
index 5682f14..28fe5c2 100644
--- a/node_modules/react-native-gifted-charts/src/BarChart/Animated2DWithGradient.tsx
+++ b/node_modules/react-native-gifted-charts/src/BarChart/Animated2DWithGradient.tsx
@@ -1,3 +1,4 @@
+//@ts-nocheck
 import React, {useEffect, useState} from 'react';
 import {View, LayoutAnimation, Platform, UIManager, Text} from 'react-native';
 import Svg, {Defs, Rect} from 'react-native-svg';
diff --git a/node_modules/react-native-gifted-charts/src/BarChart/RenderBars.tsx b/node_modules/react-native-gifted-charts/src/BarChart/RenderBars.tsx
index f06132f..f67c19a 100644
--- a/node_modules/react-native-gifted-charts/src/BarChart/RenderBars.tsx
+++ b/node_modules/react-native-gifted-charts/src/BarChart/RenderBars.tsx
@@ -1,3 +1,4 @@
+//@ts-nocheck
 import React from 'react';
 import {View, TouchableOpacity, Animated, Text} from 'react-native';
 import AnimatedThreeDBar from '../Components/AnimatedThreeDBar';
diff --git a/node_modules/react-native-gifted-charts/src/Components/common/LinearGradient.tsx b/node_modules/react-native-gifted-charts/src/Components/common/LinearGradient.tsx
index 445297c..8366725 100644
--- a/node_modules/react-native-gifted-charts/src/Components/common/LinearGradient.tsx
+++ b/node_modules/react-native-gifted-charts/src/Components/common/LinearGradient.tsx
@@ -6,6 +6,7 @@ type LinearGradientProps = {
   start?: { x: number, y: number };
   end?: { x: number, y: number };
   colors: string[];
+  children?: any;
 }
 
 let LinearGradient: React.FC<LinearGradientProps>;
